File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldParagraph.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldParagraph extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public String defaultValue;

  @Nullable
  public DefnDtoParagraph defaultVar;

  @Nullable
  public Boolean flexHeight;

  @Nullable
  public Long lineCount;

  @Nullable
  public MetaIdField lineCountFieldId;

  @Nullable
  public Long lineCountVar;

  @Nullable
  public Long maxCharCount;

  @Nullable
  public MetaIdField maxCharCountFieldId;

  @Nullable
  public Long maxCharCountVar;

  @Nullable
  public Long minCharCount;

  @Nullable
  public MetaIdField minCharCountFieldId;

  @Nullable
  public Long minCharCountVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldParagraph.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoParagraph
import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldParagraph : DefnFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: String?
  val defaultVar: DefnDtoParagraph?
  val flexHeight: Boolean?
  val lineCount: Long?
  val lineCountFieldId: MetaIdField?
  val lineCountVar: Long?
  val maxCharCount: Long?
  val maxCharCountFieldId: MetaIdField?
  val maxCharCountVar: Long?
  val minCharCount: Long?
  val minCharCountFieldId: MetaIdField?
  val minCharCountVar: Long?
}